.github/workflows/release.yml

```diff
@@ -35,7 +35,8 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.21
+          go-version-file: go.mod
```

Comment:
automatically take the version from go.mod

This will also prevent things from running out of sync like currently is the situation.